### PR TITLE
OTG HS patches and register unprefix

### PR DESCRIPTION
**Pull request by albru123**
Similarly to OTG FS fixes, this patch should improve and unify OTG register naming, fix incorrect register addresses, and remove registers that shouldn't be there and collect appropriate registers into arrays.

### CODE REVIEW

File: devices/stm32f446.yaml [hunk 1]
```diff
@@ -130,3 +130,4 @@ _include:
  - ../peripherals/fsmc/fsmc_sd.yaml
  - ../peripherals/tim/tim_ccm_v1.yaml
  - ../peripherals/sai/sai.yaml
+ - common_patches/otg_hs_remove_prefix.yaml
```
Comment (Disasm) [inline devices/stm32f446.yaml]: I don't think it's important, but I placed patch lines between `- common_patches/rtc/rtc_bkpr.yaml` and `- common_patches/tim/tim_ccr.yaml` for consistency.

Comment (albru123) [reply devices/stm32f446.yaml]: Moved :)

**Comment by Disasm** [comment]
Depends on: https://github.com/albru123/stm32-rs/pull/1

**Comment by albru123** [comment]
Merged albru123/stm32-rs#1

**Pull request merged**